frontend/src/containers/CamListContainer/CamListContainer.js

```diff
@@ -52,20 +63,42 @@ class CamListContainer extends Component {
     );
   }
 
+  getCameraDetails() {
```

Comment:
kell ez? ez maga  a CameraDetails

Reply:
ezt a getCameraDetails()-t hagytam meg egyedül a CamListContainerben külön, szerintem így átláthatóbb ami a returnben van.